software/docs.yml

```
@@ -0,0 +1,12 @@
+name: "Docs"
+website_url: "https://docs.numerique.gouv.fr/"
+source_code_url: "https://github.com/suitenumerique/docs"
+description: "A collaborative note taking, wiki and documentation platform that scales."
+licenses:
+  - MIT
+platforms:
+  - Docker
+tags:
+  - Document Management
+  - Wikis
+  - Knowledge Management Tools
```

Not to sure about the `Wikis` tag, but as far as I could see you the `Document Management` doesn't really fit here, since you can't upload documents (such as pdf or so)

```suggestion
tags:
  - Note-taking & Editors
  - Knowledge Management Tools
```

---

Fair on this one, i'll take off document management, I don't think its really a wiki per say, but the design shows that you *could* technically use it as a internal wiki, but idk. I'll remove it and we can add it back later just to play it safe.

---

📚 Built-in wiki functionality to turn your team's collaborative work into organized knowledge ETA 02/2025 as said in their roadmap